utils/field.js

```diff
@@ -453,6 +408,63 @@ function getEnumValuesWithCustomValues(enumValues) {
   });
 }
 
+function prepareClientFieldForTemplates(entityWithConfig, field, generator) {
+  if (field.fieldValidateRulesPatternAngular === undefined) {
```

Comment:
A sub function by client kind? may help for the split in the future

Reply:
Yep, split at v8.